packages/build/tests/install/snapshots/tests.mjs.md

```diff
@@ -1012,21 +1016,21 @@ Generated by [AVA](https://avajs.dev).
           package: '@netlify/plugin-local-install-core'␊
         - inputs: {}␊
           origin: config␊
-          package: ./plugin/main.js␊
+          package: ./plugin/main.mjs␊
```

Comment:
Some of the files cannot use `package.json` and need to use the file extension `.mjs` instead, which explains those change in some of the test snapshots.